src/Components/HighChartsWrapper/HighChartsWrapper.tsx

```diff
@@ -196,8 +196,7 @@ const HighChartsWrapper: React.FC<IHighChartsWrapperProps> = (props) => {
         tooltip: {
             shared: true,
             useHTML: true,
-            style: tooltipStyles?.root,
-            valueDecimals: 2
+            style: tooltipStyles?.root
```

Comment:
Maybe 2 was too small of a threshold but how many decimal places do we really need?

Reply:
Yeah, I removed it because no matter what it was adding decimals even the number is integer like 12.00. On that file on the top when I generate the highchart series data I am already doing the rounding for the data to 2 decimals. I can increase it after getting feedback from Riley